Gemfile

```diff
@@ -73,6 +73,6 @@ gem 'sorbet-runtime'
 gem 'state_machines-activerecord'
 gem 'turbo-rails', '~> 0.5.9'
 gem 'view_component', '~> 2.18'
-gem 'webpacker', '~> 5.0'
+gem 'webpacker', '6.0.0.beta7'
```

Comment:
Is the intention to pin webpacker until the version `6.0.0` is no longer in beta?

Reply:
Yes, that's correct.